_posts/release/2018-07-18-accumulo-1.9.3.md

```diff
@@ -38,10 +48,67 @@ This issue was fixed in pull request [#562]
 
 ### ~Another notable item here~ (affects ~versions~)
 
-More description here. This was fixed in [#NUM].
-
 ## Other Changes
 
+* 07d6346b1 Fix [#994] Consider mutation queue in example config [#1055]
```

Comment:
I'm very much against using an "Other changes" section to merely reproduce git history. Anything we put on the website should be curated, added because doing so will be of interest to the user, not simply as a record of the changes.

Reply:
I hear ya... I was just putting all of them on here to make it easier for us to "nominate" or bump them up into notable changes.

Reply:
Also, the link we supplied for a list of other changes for 1.9.2 was only issues and PRs with a label.  It doesn't include commits we made directly, even though these are usually insignificant or fix issues, I wanted us to start with a list of everything and whittle it down.

Reply:
If it's helpful, we could do a link like: https://github.com/apache/accumulo/compare/rel/1.9.1...apache:rel/1.9.2

Reply:
Yeah that might be better for a catch all link at the end.